src/main/java/com/symtoms/checker/alexa/handler/YesIntentHandler.java

```diff
@@ -0,0 +1,46 @@
+package com.symtoms.checker.alexa.handler;
+
+import java.util.Map;
+import java.util.Optional;
+
+import javax.annotation.Resource;
+
+import com.amazon.ask.dispatcher.request.handler.HandlerInput;
+import com.amazon.ask.model.Response;
+import com.symtoms.checker.alexa.handler.AbstractIntentHandler;
+import com.symtoms.checker.alexa.handler.HelpIntentHandler;
+
+public class YesIntentHandler  extends AbstractIntentHandler {
+
+	@Resource(name="helpHandler")
+	private HelpIntentHandler helpHandler;
+	@Resource(name="bodyLocationHandler")
+	private BodyLocationIntentHandler bodyLocationHandler;
+
+	@Override
+	public Optional<Response> handle(HandlerInput input) {
+		String speechText = "";
+		Object typeObject = getSessionAttributes(input,"type");
+		setSessionAttributes(input, "user_option", Boolean.TRUE);
+		if(null != typeObject && typeObject instanceof String) {
+			String type = (String) typeObject;
+			switch (type) {
+			case "help":
+				return helpHandler.handle(input);
+			case "launch":
+				return bodyLocationHandler.handle(input);
+			default:
+				speechText = "Sorry, I do not understand how to process that.";
```

Comment:
Better to move this validation in VM file

Reply:
Limitation of Framework